WpfSendKeys/SendKeysParser.cs

```diff
@@ -217,25 +44,60 @@ private void ParseChar()
                 key = (Key)new KeyConverter().ConvertFromInvariantString(ch);
```

Comment:
It seems like it should be safe. I guess we'll find out if we run our tests in parallel eventually.